+jrclust/+utils/metaToStruct.m

```diff
@@ -11,7 +11,7 @@
     end
 
     fid = fopen(filename, 'r');
-    keysvals = textscan(fid, '%s%s', 'Delimiter', '=',  'ReturnOnError', 0);
+    keysvals = textscan(fid, '%[^=] = %[^\r\n]',  'ReturnOnError', 0);
```

Comment:
Is this more robust? Did the other way fail on you in some way? Does it handle cases where the `=` isn't separated by spaces?